library/core/src/fmt/mod.rs

```diff
@@ -738,8 +744,10 @@ pub trait Display {
     ///     }
     /// }
     ///
-    /// assert_eq!("(1.987, 2.983)",
-    ///            format!("{}", Position { longitude: 1.987, latitude: 2.983, }));
+    /// assert_eq!(
+    ///     "(1.987, 2.983)",
+    ///     format!("{}", Position { longitude: 1.987, latitude: 2.983, }),
```

Comment:
Unrelated, but why is the longitude displayed in front of the latitude? That's pretty misleading when using tuple notation.

Reply:
I think this is the way that positions are typically stored in databases, I remember being annoyed by some GIS system because I kept reading it backwards. Agreed that it's nicer to read as `(lat, long)`.